.github/release.yml

```diff
@@ -0,0 +1,33 @@
+changelog:
+  categories:
+    - title: New Features
+      labels:
+        - feature
+    - title: Improvements
+      labels:
+        - enhancement
+        - framework
+        - tech-debt
+    - title: Security 
+      labels:
+        - security
+    - title: Bug Fixes
+      labels:
+        - bug
+    - title: Dependencies
+      labels:
+        - dependencies
+    - title: Documentation
+      labels:
+        - documentation
+    - title: Automation and Testing
+      labels:
+        - CI/CD
+        - test-only
+    - title: Breaking Changes
+      labels:
+        - major
+        - integration
```

Comment:
Can remove the integration label if we think it is only relevant internally

Reply:
We can try it for now